test-suite/build.gradle

```diff
@@ -22,8 +22,8 @@ micronautBuild {
     }
 }
 
-apply from: "${rootProject.projectDir}/gradle/geb.gradle"
-apply from: "${rootProject.projectDir}/gradle/webdriverbinaries.gradle"
+//apply from: "${rootProject.projectDir}/gradle/geb.gradle"
```

Comment:
You probably want to remove those comments

Reply:
👍